scalardb_fdw/Makefile

```diff
@@ -33,7 +33,7 @@ OS = $(shell uname | tr '[:upper:]' '[:lower:]')
 
 PG_CPPFLAGS = -I$(JAVA_HOME)/include -I$(JAVA_HOME)/include/$(OS) -D'SCALARDB_JAR_PATH=$(scalardb_jar_path)'
 
-libjvm_path = $(shell find -L $(JAVA_HOME) -name libjvm\.*)
+libjvm_path = $(word 1, $(shell find -L $(JAVA_HOME) -name libjvm\.*))
```

Comment:
Ah, sorry. I forgot to leave a comment on this. Right, this is not directly related to this PR. This is required when the JDK distribution has multiple paths to `libjvm`. For example, Azule JDK 8 on macOS provides two paths. (They are physically the same file because `jre` is a symbolic link to `zulu-8.jdk/Contents/Home/jre`.)

```console
jre/lib/server/libjvm.dylib
zulu-8.jdk/Contents/Home/jre/lib/server/libjvm.dylib
```

It is necessary for this Makefile to work properly to set a single path to `libjvm_path.`. That's why we need `word 1` to select a single path.

Previously, I used `adoptopenjdk-8`, which provides only a single path to `libjvm`, so I didn't notice this issue.